value.go

```diff
@@ -1388,7 +1388,11 @@ func (vlog *valueLog) populateDiscardStats() error {
 
 	var statsMap map[uint32]int64
 	if err := json.Unmarshal(vs.Value, &statsMap); err != nil {
-		return err
```

Comment:
now we are not returning error from here and this function is used internally. we can remove error from its return type.

Reply:
https://github.com/dgraph-io/badger/blob/e9447c910efd3c67c5453ea1f65d2f355544dd82/value.go#L1376-L1396

We return error from the `db.get()` call.